src/Uno.UI/UI/Xaml/FrameworkElement.Interface.wasm.cs

```diff
@@ -120,7 +120,32 @@ protected virtual void OnBackgroundChanged(DependencyPropertyChangedEventArgs e)
 			var brush = e.NewValue as Brush;
 			SetBackgroundBrush(brush);
 
-			_backgroundSubscription.Disposable = Brush.AssignAndObserveBrush(brush, _ => SetBackgroundBrush(brush));
+			if (brush is ImageBrush imgBrush)
+			{
+				RecalculateBrushOnSizeChanged(false);
+				_backgroundSubscription.Disposable = imgBrush.Subscribe(img =>
+				{
+					switch (img.Kind)
+					{
+						case ImageDataKind.Empty:
+						case ImageDataKind.Error:
+							ResetStyle("background-color");
+							ResetStyle("background-image");
+							break;
+
+						case ImageDataKind.Base64:
+						case ImageDataKind.Url:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty 'case' clause.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5633443)